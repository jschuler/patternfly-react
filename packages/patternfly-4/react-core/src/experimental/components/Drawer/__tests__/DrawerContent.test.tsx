// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { DrawerContent } from '../DrawerContent';

describe('DrawerContent test', () => {
  it('DrawerContent should match snapshot', () => {
    const view = shallow(
      <DrawerContent
        className={"''"}
				children={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
