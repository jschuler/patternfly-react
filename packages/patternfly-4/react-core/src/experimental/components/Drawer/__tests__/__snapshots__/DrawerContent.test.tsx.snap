// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerContent test DrawerContent should match snapshot 1`] = `
<div
  className="pf-c-drawer__content"
/>
`;
